interp/src/flatten/primitives/stateful/math.rs

```diff
@@ -387,7 +387,7 @@ impl<const DEPTH: usize> Primitive for FxpMultPipe<DEPTH> {
                             2 * (self.frac_width + self.int_width),
                         )
                         .slice(
-                            2 * self.frac_width + self.int_width,
+                            (2 * self.frac_width + self.int_width) - 1,
```

Comment:
This makes a lot of sense, since @EclecticGriffin and I discovered that the old slice implementation was off by one in how it defined the MSB.